app/models/payment.rb

```diff
@@ -9,6 +9,8 @@ def self.valid_sources
   end
 
   validates :source, presence: true, inclusion: { in: valid_sources }
-  validates :account, :amount, presence: true
+  validates :account, presence: true
+  validates :amount, presence: true, numericality: true
+  validates :processing_fee, numericality: true, allow_nil: false
```

Comment:
Ah yeah, it's `other_than` rather than `not_equal_to`.  Bummer it's not rails 3.2.  Exclusion works for now.  Or `numericality` `greater_than: 0, less_than: 0` might work as well.